apps/angular/simple-animations/src/app/paragraph.component.ts

```diff
@@ -0,0 +1,24 @@
+import { animate, style, transition, trigger } from '@angular/animations';
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+
+@Component({
+  standalone: true,
+  selector: 'app-paragraph',
+  templateUrl: 'paragraph.component.html',
+  styles: [],
+  animations: [
+    trigger('myAnimation', [
```

Comment:
You can refactor this animation code to a new `paragraph.animation.ts` file, i like this approach because it reduces the lines of code for this component.